packages/ui/app/page.tsx

```diff
@@ -26,8 +26,14 @@ function ChatHeader() {
 
 function Home() {
   return (
-    <VStack spacing="10em" justify={"center"}>
-      <Heading>Exemplo de chat</Heading>
+    <VStack 
+      spacing="10em" 
+      justify={"center"}
+      pl={4}
+      pr={4}
+      pb={6}
+      pt={6}
```

Comment:
podemos usar assim, pra simplificar:

```suggestion
      px={4}
      py={6}
```

Reply:
Resolvido no commit 5b8c6a6c0301d87ed4c3f9f8b5d8886d18bea913